datastores/gossip_kv/server/main.rs

```diff
@@ -55,19 +69,66 @@ async fn metrics_handler() -> Result<impl warp::Reply, Infallible> {
     ))
 }
 
+/// Setup serialization for outbound networking messages.
+fn setup_outbound_serialization<OUTBOUND, MESSAGE>(
+    outbound: OUTBOUND,
+) -> impl Sink<(MESSAGE, SocketAddr), Error = Error>
+where
+    OUTBOUND: Sink<(bytes::Bytes, SocketAddr), Error = Error>,
+    MESSAGE: Serialize + Debug + Send + 'static,
+{
+    outbound.with(|(msg, addr): (MESSAGE, SocketAddr)| {
+        ready(Ok::<(bytes::Bytes, SocketAddr), Error>((
+            hydroflow::util::serialize_to_bytes(msg),
+            addr,
+        )))
+    })
+}
+
+/// Setup deserialization for inbound networking messages.
+
+fn setup_inbound_deserialization<INBOUND, MESSAGE>(
+    inbound: INBOUND,
+) -> impl hydroflow::futures::Stream<Item = (MESSAGE, SocketAddr)>
+where
+    INBOUND: hydroflow::futures::Stream<Item = Result<(bytes::BytesMut, SocketAddr), Error>>,
+    MESSAGE: for<'de> serde::Deserialize<'de> + Debug + Send + 'static,
```

Comment:
Let's use PascalCase for generic type parameters

Reply:
Ah right. Fixed.